downloads/models.py

```diff
@@ -360,6 +360,7 @@ class ReleaseFile(ContentManageable, NameSlugModel):
         "SPDX-2 SBOM URL", blank=True, help_text="SPDX-2 SBOM URL"
     )
     md5_sum = models.CharField('MD5 Sum', max_length=200, blank=True)
+    sha256_sum = models.CharField('SHA256 Sum', max_length=200, blank=True)
```

Comment:
Is this user-visible text (e.g. in the table)? If so, could we say 'checksum' instead of 'sum'?:

```suggestion
    md5_sum = models.CharField('MD5 Checksum', max_length=200, blank=True)
    sha256_sum = models.CharField('SHA256 Checksum', max_length=200, blank=True)
```

Reply:
will do it in release_detail page , but thanks. this helped me see we'd be missing the frontend bits